src/coalescenceml/integrations/azure_datalake/artifact_store/azure_artifact_store.py

```diff
@@ -0,0 +1,83 @@
+from ast import Tuple
+import os
+from pathlib import Path
+from adlfs import AzureBlobFileSystem
+
+from coalescenceml.artifact_store.base_artifact_store import BaseArtifactStore, PathType
+from typing import (
+    Any,
+    Callable,
+    Iterable,
+    List,
+    Optional,
+)
+
+class AzureArtifactStore(BaseArtifactStore):
+    """ Artifact Store for Azure Datalake artifacts""" 
+
+    abfs: Optional[AzureBlobFileSystem] = None
+    account_name: str
+
```

Comment:
This is missing the `SUPPORTED_SCHEMES` attribute. For Blob I believe there are `”abfs://”` and `”as://”`.